src/NuGet.ProjectModel/NuGet.ProjectModel.xproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
   <Import Project="$(VSToolsPath)\AspNet\Microsoft.Web.AspNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>f013e43f-b6d5-4f59-acf0-eecec2c794f5</ProjectGuid>
+    <ProjectGuid>18006692-afcc-49fd-95f6-66e8a0c49e9b</ProjectGuid>
```

Comment:
VS keeps insisting I change the GUID... fine. Maybe checking it in will shut it up :)